Another-Game-Engine/AGE
# Texture compression microsoft lib

Some instructions to test this PR:
- Download the Microsoft DirectXTex lirary (https://directxtex.codeplex.com/releases/view/615824) and put it in the vendors folder
- Download the assets on the google drive (https://drive.google.com/file/d/0BwlWFYW_NU-Oa3FLZmxQd0VlRWM/view?usp=sharing)
- If you want to be able to see the converted textures, download a dds viewer (http://www.nvidia.co.uk/object/windows_texture_viewer.html)
- If you want to create some cubemaps/skyboxes, download the AMD cubemap generator (http://developer.amd.com/tools-and-sdks/archive/legacy-cpu-gpu-tools/cubemapgen/)
